src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1439,13 +1439,20 @@ private static bool CanCreateNodeFromSelection(object parameter)
         }
 
         /// <summary>
-        /// Gets the selected "input" nodes
+        /// Return all nodes from the current selection whose
+        /// IsInputNode property is true or null.
```

Comment:
this doesn't seem accurate? - we only return nodes that are null if they are code blocks.